src/ExNavigationStack.js

```diff
@@ -435,6 +435,16 @@ class ExNavigationStack extends PureComponent<any, Props, State> {
     );
   }
 
+  _getNavigationBarHeight(latestRouteConfig) {
+    let height = NavigationBar.DEFAULT_HEIGHT;
+
+    if (latestRouteConfig.statusBar && latestRouteConfig.statusBar.translucent) {
+      height = NavigationBar.DEFAULT_HEIGHT_WITHOUT_STATUS_BAR + 24;
```

Comment:
yeah. @brentvatne defined it at 24 already when in exponent env. I just mirrored that.

also some reason 25 had a weird spacing issue.